### PR TITLE
Update PF_Footer.feature

### PR DESCRIPTION
**Pull request by sahajisy**
Updated Feature file with @Smoke Tags and Added @Full Tag

```
@FooterPage @Full
Feature: Validate footer links on Footer Sections

#Testcase no: PT-090 to 95
@Smoke
Scenario: Validate Footer Links on Top Page

Then Check footer links on Top Page

#Testcase no: PT-122
@Smoke
Scenario: Check Links on Terms and Conditions Page

Then Check links in Terms of Use Page

#Testcase no: PT-102 to PT-105
@Smoke
Scenario: Validate Footer on Account and Services Page 
Then Check footer links on Login Page

#Testcase no: PT-096 to PT-101
@Smoke
Scenario: Validate Footer on User Top Page 

Given Login to Platform Portal
Then Check footer links on top page after login

#Testcase no: PT-102 to PT-105
@Smoke
Scenario: Validate Footer on Order Screen One

Given Login to Platform Portal
Then Check Footer Links on First Order Screen

#Testcase no: PT-102 to PT-105
@Smoke
Scenario: Validate Footer on Order Screen Two and Three

Given Login to Platform Portal
Then Check Footer Links on Second Order Screen
Then Check Footer Links on Third Order Screen
```

### CODE REVIEW

File: platform/src/test/java/platformfeatures/PF_Footer.feature [hunk 1]
```diff
@@ -1,33 +1,39 @@
-@FooterPage @Smoke
+@FooterPage @Full
 Feature: Validate footer links on Footer Sections
 
 #Testcase no: PT-090 to 95
+@Smoke
```
Comment (csonparote) [inline platform/src/test/java/platformfeatures/PF_Footer.feature]: You can add @Smoke tag before @Full if u need to test all scenarios

**Review by csonparote — CHANGES_REQUESTED** [review]
Please make changes if possible

File: platform/src/test/java/platformfeatures/PF_Footer.feature (hunk 1 above)
Comment (sahajisy) [reply platform/src/test/java/platformfeatures/PF_Footer.feature]: @csonparote , I have added @Smoke tag before @Full.

**Review by csonparote — APPROVED** [review]
Plz merge into main if executing as intended

**Pull request merged**